iphone/TitaniumKit/TitaniumKit/Sources/API/TiUIView.h

```diff
@@ -82,6 +82,7 @@ void ModifyScrollViewForKeyboardHeightAndContentHeightWithResponderRect(UIScroll
   UITapGestureRecognizer *doubleTapRecognizer;
   UITapGestureRecognizer *twoFingerTapRecognizer;
   UIPinchGestureRecognizer *pinchRecognizer;
+  UIRotationGestureRecognizer *rotationRegognizer;
```

Comment:
```suggestion
  UIRotationGestureRecognizer *rotationRecognizer;
```

Please also fix in all usages